store/etcd/types_transformation.go

```diff
@@ -0,0 +1,182 @@
+package etcd
```

Comment:
:tulip: I suggest renaming this file to something like `encoding.go`, wdyt?

Reply:
I agree that we can do better with the name of this file but I am not sure that `encoding.go` is the more appropriate one. We are not doing any encoding there. We are just creating new objects type based on others.
I am sure that you can offer something more "beau gosse".

Reply:
Maybe simply `proto.go`? This shows that among the other files, this is the one that deals with protobuf in the `etcd` package.

Reply:
Done.
I also renamed `proto/proto.go` to `proto/proto_gen.go`.